Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -591,6 +591,8 @@ class WebExtensionContext : public API::ObjectImpl<API::Object::Type::WebExtensi
     void addInjectedContent(WebUserContentControllerProxy&);
     void removeInjectedContent(WebUserContentControllerProxy&);
 
+    void addDeclarativeNetRequestRulesToUserContentController(WebUserContentControllerProxy&);
```

Comment:
```suggestion
    void addDeclarativeNetRequestRules(WebUserContentControllerProxy&);
```

I think we also need a `removeDeclarativeNetRequestRules()` that is called by `WebExtensionController::removeUserContentController()`.